eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -191,16 +191,7 @@ function Set-ChangeLogContent {
   $changeLogContent += "# Release History"
   $changeLogContent += ""
 
-  try
-  {
-    $VersionsSorted = [AzureEngSemanticVersion]::SortVersionStrings($ChangeLogEntries.Keys)
```

Comment:
OK this is is what removed some of the entries that didn't match? So we no longer sort the entries at all then? That might be easier but I just want to confirm.

Reply:
Yes, we were sorting it in three places. I removed two of them. We still do one sorting [here](https://github.com/Azure/azure-sdk-tools/blob/965697c69ea541bfdab23ac0e80b41cab0ad8e44/eng/common/scripts/Update-ChangeLog.ps1#L90). I left that one because it shouldn't be too much of a problem, am just using it to see if an older version than the latest is being used for the update.